man/fi_mr.3.md

```diff
@@ -800,6 +800,12 @@ configure registration caches.
   are: 0 or 1. Note that the ROCR memory monitor requires a ROCR version with
   unified virtual addressing enabled.
 
+*FI_MR_ZE_CACHE_MONITOR_ENABLED*
+: The ZE cache monitor is responsible for detecting ZE device memory
+  (FI_HMEM_ZE) changes made between the device virtual addresses used by an
+  application and the underlying device physical pages. Valid monitor options
+  are: 0 or 1.
```

Comment:
Does ZE support the CUDA/ROCR equivalent of unified virtual addressing?

Reply:
Yes, that's always the case according to the documentation: https://spec.oneapi.com/level-zero/latest/core/PROG.html#memory-and-images

Reply:
Thanks. I assumed all GPU vendors moved to this approach but just had to verify.